components/soyosource_virtual_meter/soyosource_virtual_meter.cpp

```diff
@@ -20,6 +20,11 @@ void SoyosourceVirtualMeter::setup() {
              this->last_power_demand_);
 
     this->last_power_demand_received_ = millis();
+
+    //if update intervall is set to never, trigger the update() method from here
```

Comment:
```suggestion
    // If the update_interval is set to never, call the update() method on every power sensor update
```

Reply:
good suggestion, can't commit it right now. I guess you need to update it.